keyboards/dz60/keymaps/mekberg/keymap.c

```diff
@@ -0,0 +1,102 @@
+#include QMK_KEYBOARD_H
+
+/*
+DZ60 ISO physical layout
+1u == 8chars
+,-------------------------------------------------------------------------------------------------------------------------------------.
+|        |        |        |        |        |        |        |        |        |        |        |        |        |      2u        |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|    1,5u    |        |        |        |        |        |        |        |        |        |        |        |        |     1,5u   |
+|--------------------------------------------------------------------------------------------------------------------------|          |
+|     1,75u    |        |        |        |        |        |        |        |        |        |        |        |        |   1,25u  |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|   1,25u  |        |        |        |        |        |        |        |        |        |        |        |         2,75u         |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|   1,25u  |   1,25u  |   1,25u  |                                                        |   1,25u  |   1,25u  |   1,25u  |   1,25u  |
+`-------------------------------------------------------------------------------------------------------------------------------------'
+*/
+
+// Fillers to make layering more clear
+#define XXXXXX KC_TRNS
+#define ______ KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

Reply:
Actually, the reason I include them is because I flip them around. To me, they make more sense the other way when written in this "ascii art" format. Having "XXXXXX" all over makes it so much harder to see actual keycode names in the map.